eslint.config.js

```diff
@@ -1,22 +1,32 @@
+// @ts-check
 import js from '@eslint/js'
 import ts from 'typescript-eslint'
+import tseslint from 'typescript-eslint'
```

Comment:
uhh; I should probably combine these two imports, this is silly.
- [ ] TODO: fix this.

Reply:
😂 <3